netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -249,68 +279,67 @@ public void inboundDataShouldForwardToStream() throws Exception {
     // Create a data frame and then trigger the handler to read it.
     // Need to retain to simulate what is done by the stream.
     ByteBuf frame = dataFrame(3, false).retain();
-    handler.channelRead(this.ctx, frame);
+    handler.channelRead(ctx, frame);
     verify(stream).transportDataReceived(eq(content), eq(false));
   }
 
   @Test
-  public void createShouldQueueStream() throws Exception {
-    // Disallow stream creation to force the stream to get added to the pending queue.
+  public void createShouldBufferStream() throws Exception {
+    // Force the stream to be buffered.
     setMaxConcurrentStreams(0);
-    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream),
-        promise);
-
-    // Make sure the write never occurred.
-    verify(frameListener, never()).onHeadersRead(eq(ctx),
-        eq(3),
-        any(Http2Headers.class),
-        eq(0),
-        eq(Http2CodecUtil.DEFAULT_PRIORITY_WEIGHT),
-        eq(false),
-        eq(0),
-        eq(false));
+    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream), promise);
+    verifyNoMoreInteractions(ctx);
   }
 
   @Test
-  public void receivedGoAwayShouldFailQueuedStreams() throws Exception {
-    // Force a stream to get added to the pending queue.
+  public void receivedGoAwayShouldFailBufferedStream() throws Exception {
+    // Force the stream to be buffered.
     setMaxConcurrentStreams(0);
-    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream),
-        promise);
-
+    ChannelPromise promise = new DefaultChannelPromise(channel, ImmediateEventExecutor.INSTANCE);
+    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream), promise);
     handler.channelRead(ctx, goAwayFrame(0));
-    verify(promise).setFailure(any(Throwable.class));
+    assertTrue(promise.isDone());
+    assertFalse(promise.isSuccess());
+    verify(stream).transportReportStatus(any(Status.class), eq(true),
+            notNull(Metadata.Trailers.class));
   }
 
   @Test
   public void receivedGoAwayShouldFailUnknownStreams() throws Exception {
-    // Force a stream to get added to the pending queue.
-    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream),
-        promise);
+    handler.write(ctx, new CreateStreamCommand(grpcHeaders, stream), promise);
 
     // Read a GOAWAY that indicates our stream was never processed by the server.
-    handler.channelRead(ctx,
-        goAwayFrame(0, 8 /* Cancel */, Unpooled.copiedBuffer("this is a test", UTF_8)));
+    handler.channelRead(ctx, goAwayFrame(0, 8 /* Cancel */,
```

Comment:
I get I bit annoyed when there are _unrelated_ whitespace changes in a large PR, as it makes it much harder to review (FYI), but it is particularly aggravating when they aren't necessarily positive changes. Unrelated changes in general are a recipe for everyone getting upset because they have a history of blocking the core of a PR; the reviewer is annoyed by the noise and blocking something they want because of changes that were not necessary and the author is annoyed by the delay and "nits".

In this case [wrapping should prefer breaking at a **higher syntactic level**](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.5.1-line-wrapping-where-to-break), so the previous wrapping could be argued to be better.